tests/generator/test_rom.py

```diff
@@ -142,6 +144,12 @@ def test_get_name_puzzle_conditions(self, softfork_height: int) -> None:
             birth_seconds=None,
             create_coin=[(bytes([0] * 31 + [1]), 500, None)],
             agg_sig_me=[],
+            agg_sig_parent=[],
```

Comment:
we should probably make all of these keyworded at some point.

Reply:
every time I have to update this, I think to myself: "this should be factored out". Maybe the next time it happens it will be painful enough to actually fix it